pjmedia/src/pjmedia/transport_udp.c

```diff
@@ -1218,11 +1215,8 @@ static pj_status_t transport_media_stop(pjmedia_transport *tp)
     if (!udp->started)
 	return PJ_SUCCESS;
 
-    pj_ioqueue_post_completion(udp->rtp_key, &udp->rtp_read_op,
-			       -PJ_ECANCELLED);
-
-    pj_ioqueue_post_completion(udp->rtcp_key, &udp->rtcp_read_op,
-			       -PJ_ECANCELLED);
+    pj_ioqueue_clear_key(udp->rtp_key);
```

Comment:
With this, app will no longer receive callback with cancelled status. Wondering if this can cause any problem, such as in app that expects to receive such callback.

Reply:
Currently the using app is just UDP media transport and I think it is fine without callback.

Moreover, callback is bound to operation key, so there should be two callbacks if there are two outstanding ops and for the infinite issue, where the operation lists seem to be in bad state, we cannot iterate the list.

Reply:
Perhaps add an option to maintain the old behavior?

Reply:
The modification only affects UDP media transport internally, there should not be any behavior change for app.